titanic.ipynb

```diff
@@ -8,6 +8,7 @@
         "## Titanic 3\n",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 1 Line 1_
```markdown
## Titanic 3
```
titanic 3 review comment 3

_Respond and view the context [here](https://gitnotebooks.com/smith-kyle/notebook-diff-tests/pull/10)._

Reply:
3 sec